File: qc-core/src/main/java/com/eager/questioncloud/core/domain/review/event/ModifiedReviewEvent.java
package com.eager.questioncloud.core.domain.review.event;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class ModifiedReviewEvent {
    private Long questionId;
    private int varianceRate;

    public static ModifiedReviewEvent create(Long questionId, int varianceRate) {
        return new ModifiedReviewEvent(questionId, varianceRate);
    }
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/review/event/ModifiedReviewEvent.kt
package com.eager.questioncloud.core.domain.review.event

class ModifiedReviewEvent(
    val questionId: Long,
    val varianceRate: Int,
) {
    companion object {
        @JvmStatic
        fun create(questionId: Long, varianceRate: Int): ModifiedReviewEvent {
            return ModifiedReviewEvent(questionId, varianceRate)
        }
    }
}
